### DISCUSSION TITLE
My MacBook suddenly won't connect to github.com.

### DISCUSSION THREAD

**Discussion by nohjiho** [community/community#148009] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was able to access GitHub on my MacBook, but at some point I couldn't connect.

Has anyone had the same experience?

I'm using a MacBook and a Windows laptop. They're connected to the same Wi-Fi.

I can connect on my Windows laptop, but not on my MacBook, so I'm asking for help.

1. Chrome browser developer mode network capture
<img width="1678" alt="스크린샷 2024-12-27 오후 5 55 14" src="https://github.com/user-attachments/assets/32b35194-e58c-4e0f-a603-9de3cc2c2b2a" />

2. Below is the network access test log of https://github-debug.com/ on a MacBook.
1st execution result

mac@macui-MacBookPro vivabutton_job_app % GIT_TRACE=1 GIT_TRANSFER_TRACE=1 GIT_CURL_VERBOSE=1 git clone https://github.com/github/debug-repo /tmp/debug-repo-http
17:18:54.185635 git.c:463 trace: built-in: git clone https://github.com/github/debug-repo /tmp/debug-repo-http
'/tmp/debug-repo-http'에 복제합니다...
17:18:54.204140 run-command.c:659 trace: run_command: git remote-https origin https://github.com/github/debug-repo
17:18:54.213891 git.c:749 trace: exec: git-remote-https origin https://github.com/github/debug-repo
17:18:54.214467 run-command.c:659 trace: run_command: git-remote-https origin https://github.com/github/debug-repo
17:18:54.232778 http.c:820 == Info: Couldn't find host [github.com](https://github.com/) in the .netrc file; using defaults
17:18:54.235241 http.c:820 == Info: Host [github.com:443](https://github.com/) was resolved.
17:18:54.235255 http.c:820 == Info: IPv6: (none)
17:18:54.235259 http.c:820 == Info: IPv4: 20.200.245.247
17:18:54.235287 http.c:820 == Info: Trying 20.200.245.247:443...
17:20:09.237418 http.c:820 == Info: connect to 20.200.245.247 port 443 from 192.168.45.201 port 52074 failed: Operation timed out
17:20:09.237446 http.c:820 == Info: Failed to connect to [github.com](https://github.com/) port 443 after 75003 ms: Couldn't connect to server
17:20:09.237465 http.c:820 == Info: Closing connection
fatal: unable to access 'https://github.com/github/debug-repo/': Failed to connect to [github.com](https://github.com/) port 443 after 75003 ms: Couldn't connect to server
mac@macui-MacBookPro vivabutton_job_app % GIT_TRACE=1 GIT_TRANSFER_TRACE=1 GIT_CURL_VERBOSE=1 git clone [git@github.com](mailto:git@github.com):github/debug-repo /tmp/debug-repo-ssh
17:20:50.427793 git.c:463 trace: built-in: git clone [git@github.com](mailto:git@github.com):github/debug-repo /tmp/debug-repo-ssh
'/tmp/debug-repo-ssh'에 복제합니다...
17:20:50.440483 run-command.c:659 trace: run_command: unset GIT_DIR; GIT_PROTOCOL=version=2 ssh -o SendEnv=GIT_PROTOCOL [git@github.com](mailto:git@github.com) 'git-upload-pack '''github/debug-repo''''
The authenticity of host '[github.com](https://github.com/) (20.200.245.247)' can't be established.
ED25519 key fingerprint is SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU.
This key is not known by any other names.
Are you sure you want to continue connecting (yes/no/[fingerprint])? ㅛㅛ yes
Warning: Permanently added '[github.com](https://github.com/)' (ED25519) to the list of known hosts.
[git@github.com](mailto:git@github.com): Permission denied (publickey).
fatal: 리모트 저장소에서 읽을 수 없습니다

올바른 접근 권한이 있는지, 그리고 저장소가 있는지
확인하십시오.
mac@macui-MacBookPro vivabutton_job_app % GIT_TRACE=1 GIT_TRANSFER_TRACE=1 GIT_CURL_VERBOSE=1 git clone [git@github.com](mailto:git@github.com):github/debug-repo /tmp/debug-repo-ssh
17:21:24.391694 git.c:463 trace: built-in: git clone [git@github.com](mailto:git@github.com):github/debug-repo /tmp/debug-repo-ssh
'/tmp/debug-repo-ssh'에 복제합니다...
17:21:24.403147 run-command.c:659 trace: run_command: unset GIT_DIR; GIT_PROTOCOL=version=2 ssh -o SendEnv=GIT_PROTOCOL [git@github.com](mailto:git@github.com) 'git-upload-pack '''github/debug-repo''''
[git@github.com](mailto:git@github.com): Permission denied (publickey).
fatal: 리모트 저장소에서 읽을 수 없습니다

올바른 접근 권한이 있는지, 그리고 저장소가 있는지
확인하십시오.
mac@macui-MacBookPro vivabutton_job_app % ping -c 10 [github.com](https://github.com/)
PING [github.com](https://github.com/) (20.200.245.247): 56 data bytes
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=0 ttl=113 time=3.597 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=1 ttl=113 time=9.516 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=2 ttl=113 time=6.230 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=3 ttl=113 time=3.600 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=4 ttl=113 time=10.298 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=5 ttl=113 time=9.891 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=6 ttl=113 time=3.491 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=7 ttl=113 time=3.994 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=8 ttl=113 time=4.274 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=9 ttl=113 time=9.519 ms

--- [github.com](https://github.com/) ping statistics ---
10 packets transmitted, 10 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 3.491/6.441/10.298/2.851 ms
mac@macui-MacBookPro vivabutton_job_app % traceroute [github.com](https://github.com/)
traceroute to [github.com](https://github.com/) (20.200.245.247), 64 hops max, 40 byte packets
1 192.168.45.1 (192.168.45.1) 2.189 ms 1.503 ms 1.472 ms
2 * * *
3 100.71.50.233 (100.71.50.233) 3.596 ms 2.858 ms 3.100 ms
4 10.44.255.236 (10.44.255.236) 2.690 ms
10.44.255.232 (10.44.255.232) 2.952 ms
10.44.255.238 (10.44.255.238) 6.551 ms
5 10.222.23.56 (10.222.23.56) 2.959 ms
10.222.23.52 (10.222.23.52) 3.969 ms
10.222.24.104 (10.222.24.104) 2.567 ms
6 10.222.31.39 (10.222.31.39) 3.674 ms
10.222.6.19 (10.222.6.19) 3.896 ms
10.222.31.33 (10.222.31.33) 4.156 ms
7 39.120.4.149 (39.120.4.149) 4.134 ms
39.120.4.151 (39.120.4.151) 3.784 ms
39.120.4.149 (39.120.4.149) 3.484 ms
8 [ae29-0.icr02.sel21.ntwk.msn.net](http://ae29-0.icr02.sel21.ntwk.msn.net/) (104.44.239.244) 6.417 ms
[ae22-0.icr01.sel21.ntwk.msn.net](http://ae22-0.icr01.sel21.ntwk.msn.net/) (104.44.42.135) 6.417 ms
[ae29-0.icr02.sel21.ntwk.msn.net](http://ae29-0.icr02.sel21.ntwk.msn.net/) (104.44.239.244) 9.250 ms
9 * * *
10 * * *
11 * * *
12 * * *
13 * * *
14 * * *
15 * * *
16 * * *
17 * * *
18 * * *
19 * * *
20 * * *
21 * * *
22 * * *
23 * * *
24 * * *
25 * * *
26 * * *
27 * * *
28 * * *
29 * * *
30 * * *
31 * * *
32 * * *
33 * * *
34 * * *
35 * * *
36 * * *
37 * * *
38 * * *
39 *^C
mac@macui-MacBookPro vivabutton_job_app % curl -s -o/dev/null -w "downloadspeed: %{speed_download} | dnslookup: %{time_namelookup} | connect: %{time_connect} | appconnect: %{time_appconnect} | pretransfer: %{time_pretransfer} | starttransfer: %{time_starttransfer} | total: %{time_total} | size: %{size_download}\n" https://github.com/
downloadspeed: 0 | dnslookup: 0.002202 | connect: 0.000000 | appconnect: 0.000000 | pretransfer: 0.000000 | starttransfer: 0.000000 | total: 75.003252 | size: 0

2번 실행 결과
Error downloading from [github.com](https://github.com/)
2507998 bytes downloaded from [cloud.githubusercontent.com](http://cloud.githubusercontent.com/) at 8.97 Mbps
2507998 bytes downloaded from [avatars.githubusercontent.com](http://avatars.githubusercontent.com/) at 79.40 Mbps
2507998 bytes downloaded from [github.githubassets.com](http://github.githubassets.com/) at 13.98 Mbps
2507998 bytes downloaded from [australiaeast.github-debug.com](http://australiaeast.github-debug.com/) at 10.02 Mbps
2507998 bytes downloaded from [brazilsouth.github-debug.com](http://brazilsouth.github-debug.com/) at 4.74 Mbps
2507998 bytes downloaded from [centralindia.github-debug.com](http://centralindia.github-debug.com/) at 8.50 Mbps
2507998 bytes downloaded from [fra.github-debug.com](http://fra.github-debug.com/) at 7.31 Mbps
2507998 bytes downloaded from [iad.github-debug.com](http://iad.github-debug.com/) at 6.73 Mbps
2507998 bytes downloaded from [japaneast.github-debug.com](http://japaneast.github-debug.com/) at 27.69 Mbps
2507998 bytes downloaded from [koreacentral.github-debug.com](http://koreacentral.github-debug.com/) at 41.24 Mbps
2507998 bytes downloaded from [sea.github-debug.com](http://sea.github-debug.com/) at 10.92 Mbps
2507998 bytes downloaded from [southafricanorth.github-debug.com](http://southafricanorth.github-debug.com/) at 4.66 Mbps
2507998 bytes downloaded from [southeastasia.github-debug.com](http://southeastasia.github-debug.com/) at 17.67 Mbps
2507998 bytes downloaded from [github-cloud.s3.amazonaws.com](http://github-cloud.s3.amazonaws.com/) at 9.66 Mbps

58.123.36.147

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/[131.0.0.0](http://131.0.0.0/) Safari/537.36

e148c7d9092c2fa297bca12e77f75b69

C4AD:2D5677:7453:9EC4E:676E61A9

traceroute to 58.123.36.147 (58.123.36.147), 30 hops max, 60 byte packets
1 *
2 *
3 *
4 *
5 *
6 *
7 *
8 *
9 *
10 *
11 *
12 *
13 *
14 *
15 *
16 *
17 *
18 *
19 *
20 *
21 *
22 *
23 *
24 *
25 *
26 *
27 *
28 *
29 *
30 *

PING 58.123.36.147 (58.123.36.147) 56(84) bytes of data.

--- 58.123.36.147 ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2041ms

Start: 2024-12-27T00:14:57-0800
HOST: 6181ca006a3ed04ddb24ba1f0584d689 Loss% Snt Last Avg Best Wrst StDev

Host 147.36.123.58.in-addr.arpa. not found: 3(NXDOMAIN)

Thank you in advance.

**Comment by shinybrightstar** [comment]
Hi @nohjiho,

Are you still having this problem? If not, you could close the discussion.
